pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -22,6 +22,10 @@ const (
 	// AnnotationBuildRefSecret is an annotation that tells the Build Controller to reconcile on
 	// events of the secret only if is referenced by a Build in the same namespace
 	AnnotationBuildRefSecret = "build.build.dev/referenced.secret"
+
+	// AnnotationBuildVerifyRepository is a label key that tells the Build Controller to check whether remote repository exists or not.
+	// If its value is true, controller checks remote repository, otherwise, it won't be checked
```

Comment:
Based on what I see which `validateSourceURL` we will validate by default, i.e. if the annotation is *NOT* set.  Perhaps that was a switch based on feedback ?

In any event, this Godoc should be a little more clear/precise on that ... an alternative wording:  `If the annotation is not set or has a value of 'true', the controller checks the remote repository.  A value of 'false' means the controller will bypass checking  the remote repository.`

Reply:
@gabemontero correct, based on the feedback we decided to have this enable by default, and we can always opt-out from it by setting the related annotation to `false`.

We adopted your proposed alternative wording, thanks a lot for that, please double check our changes.